accounting/fiscal_localizations/overview/fiscal_localization_packages.rst

```diff
@@ -0,0 +1,47 @@
+============================
+Fiscal Localization Packages
+============================
+
+**Fiscal Localization Packages** are country-specific modules that install pre-configured taxes,
+fiscal positions, chart of accounts, and legal statements on your database. Some additional features,
+such as the configuration of specific certificates, are also added to your Accounting app, following
+your fiscal administration requirements.
+
+.. note::
+   Odoo continuously adds new localizations and improves the existing packages.
+
+Configuration
+=============
+
+Odoo automatically installs the appropriate package for your company, according to the country
+selected at the creation of the database.
+
+.. important::
+   As long as you haven't posted any entry, you can still add and select another package.
+
+To install a new package, go to :menuselection:`Accounting --> Configuration --> Fiscal
+Localization`, click on **Install More Packages**, and install your country's module.
+
+.. image:: media/fiscal_localization_packages_modules.png
+   :align: center
+   :alt: Install the appropriate module as fiscal localization package in Odoo Accounting.
+
+Once done, select your country's package, and click on *Save*.
+
+.. image:: media/fiscal_localization_packages_selection.png
+   :align: center
+   :alt: Select your country's fiscal localization package in Odoo Accounting.
+
+Use
+===
+
+These packages require you to fine-tune your chart of accounts according to your needs, activate the
+taxes you use and configure your country-specific statements and certifications.
+
+Please refer to the documentation listed in the *see also* section for more information.
+
+.. seealso::
+   - :doc:`../../overview/getting_started/chart_of_accounts`
+   - :doc:`../../fiscality/taxes/create`
+
+.. todo:: update list of linked docs, and link with the future one about country-specific statements.
```

Comment:
Don't forget to do it! :smile: :laughing:

Reply:
This one is for a later doc that I still have to write. So I'll just run a search in my folder to find any  :todo: to do 👍

Reply:
Haha, I know that you will do it, it's just that I found it funny to remind you to not forget to do something you specified as "todo"